test/integration/test_payroll_process_scenarios/classic.js

```diff
@@ -0,0 +1,567 @@
+       cumul_annuel : 6887680,
+       cumul_mensuel : 573973,
+       taxe_ipr_id : 1,
+     },
+     {
+       id : 10,
+       currency_id : 1,
+       rate : 40,
+       tranche_annuelle_debut : 22956000,
+       tranche_annuelle_fin : 100000000000000,
+       tranche_mensuelle_debut : 1913000,
+       tranche_mensuelle_fin : 1913000,
+       ecart_annuel : 0,
+       ecart_mensuel : 0,
+       impot_annuel : 0,
+       impot_mensuel : 0,
+       cumul_annuel : 6887680,
+       cumul_mensuel : 573973,
+       taxe_ipr_id : 1,
+     }],
+    },
+  };
+
+  const paramsConfig = {
+    currency : '2',
+    employees : '0D68B9D64D1F45EDA43302D619F7CE98',
+    idPeriod : params.periodPaie,
+    lang : 'fr',
+    payslip : 'true',
+    renderer : 'json',
+  };
+
+  const paramsMultiConfig = {
+    currency : 2,
+    employees : [
+      'B8C3674B2F264B598C16FBA76F5116D1',
+      '51D6A9297B464327B6EB8155BDBDC9E0',
+    ],
+    idPeriod : params.periodPaie,
+    lang : 'fr',
+    payslip : 'true',
+    renderer : 'json',
+  };
+
+  // Amelia Rose Thornton
+  it('POST /multiple_payroll/:id/ Payroll Management: Continuity of Work For Employees Amelia Rose Thornton', () => {
+    return agent.post('/multiple_payroll/'.concat(params.periodPaie, '/configuration'))
+      .send(dataConfiguration)
+      .then((res) => {
+        expect(res).to.have.status(201);
+
+        return agent.get('/reports/payroll/payslip')
+          .query(paramsConfig);
+      })
+      .then((res) => {
+        // The seniority bonus is calculated automatically by the system, using the formula:
+        // Base Salary × Seniority Rate × Years of Service
+
+        expect(res.body.rubrics[2].total).to.equal(330.75);
+
+        // ### Breakdown of Taxable Earnings
+        // | Description               | Amount   |
+        // |---------------------------|----------|
+        // | School Fees               | $50.00   |
+        // | Cost of Living Allowance  | $0.00    |
+        // | Seniority Bonus           | $330.75  |
+        // | Other Bonuses             | $25.00   |
+        // | Taxable Net Income        | $405.75  |
+
+        expect(res.body.dataEmployees[0].somRubTaxable_equiv).to.equal(405.75);
+
+        // INSS QPO (3.5%): `(225 + 405.75) × 0.035 = $22.08
+        expect(res.body.rubrics[8].total).to.equal(22.08);
+
+        // Housing Benefit (30% of base)
+        // 225 × 0.3 = $67.5
+        expect(res.body.rubrics[5].total).to.equal(67.50);
+
+        // Breakdown of Non-Taxable Earnings
+        // | Description                           | Amount   |
+        // |---------------------------------------|----------|
+        // | Family Allowance                      | $30.00   |
+        // | In-Kind Benefit – Housing (30%)       | $67.50   |
+        // | Transport Allowance                   | $70.00   |
+        // | Non-Taxable Net Income                | $167.50  |
+        expect(res.body.dataEmployees[0].somRubNonTaxable_equiv).to.equal(167.5);
```

Comment:
It is good to check the totals here, but can you also check each item? If the total ever gets thrown off, we'll want to know where the issue is coming from.